### ISSUE TITLE
convert from python file to jupyter notebook

### ISSUE THREAD

**Issue by Sandy4321** [microsoft/vscode-jupyter#4495]
in this video 
https://www.youtube.com/watch?v=FSdIoJdSnig&ab_channel=MicrosoftVisualStudio

mentioned that there is the way to convert from python file to jupyter notebook
but how to do this ?

**Comment by DavidKutu** [comment]
Hi @Sandy4321, on the notebook toolbar look for this icon (the tooltip says 'Export As'):
![image](https://user-images.githubusercontent.com/20347396/105781030-95b51280-5f26-11eb-8d54-a8394e802768.png)

After you click it, an input box will appear with 3 options: 
1. Python Script
2. PDF
3. HTML

Select Python script and your python file will be created.

You might need to install nbconvert like this: `pip install nbconvert` (assuming you're using pip).
Let me know if it works for you.

**Comment by DavidKutu** [comment]
Hey @Sandy4321, I just realized I got your question backwards. (I'll leave my previews comment in case someone else finds it)

To convert from Python file to notebook:
1. Open the python file you want to convert
2. Right click anywhere on the python file
3. Click on 'Export Current Python File as Jupyter Notebook' or 'Export Current Python File and Output as Jupyter Notebook'
4. A Save as window will appear, name your new notebook and click 'Export'
